tests/dialects/test_athena.py

```diff
@@ -337,3 +337,22 @@ def test_parse_partitioned_by_returns_iceberg_transforms(self):
         assert isinstance(parsed.this, exp.Schema)
         assert next(n for n in parsed.this.expressions if isinstance(n, exp.PartitionedByBucket))
         assert next(n for n in parsed.this.expressions if isinstance(n, exp.PartitionByTruncate))
+
+    def test_ctas_uses_string_for_format(self):
```

Comment:
Can we add a `validate_all` test as well to demonstrate that Spark -> Athena is now covered?